x/posts/keeper/msg_server_test.go

```diff
@@ -1170,7 +1170,7 @@ func (suite *KeeperTestSuite) TestMsgServer_AddPostAttachment() {
 					gomock.Any(),
 					uint64(1),
 					uint32(0),
-					"cosmos1r9jamre0x0qqy562rhhckt6sryztwhnvhafyz4",
+					"cosmos13t6y2nnugtshwuy0zkrq287a95lyy8vzleaxmd",
```

Comment:
We should ensure the the owner of post stored is the same as the sender without EditOwnContent permission in this test.